packages/postcss-normalize-unicode/src/index.js

```diff
@@ -40,15 +43,34 @@ function unicode (range) {
     return range;
 }
 
+/*
+ * IE ignores the unicode-range if the 'U' is lowercase
+ *
+ * https://caniuse.com/#search=unicode-range
+ */
+
+function hasLowerCaseUPrefixBug (browser) {
+    return ~['ie 8', 'ie 9', 'ie 10', 'ie 11'].indexOf(browser);
```

Comment:
What about edge?

Reply:
@vitaliymaz can you great easy gist i test this using browserstack tomorrow?

Reply:
Yes, I will.

Reply:
I'm not able to work on this today. I will continue everything tomorrow.